ert3/workspace/_workspace.py

```diff
@@ -24,7 +24,20 @@ def assert_experiment_exists(workspace_root, experiment_name):
         )
 
 
-def experiment_have_run(workspace_root, experiment_name):
+def get_experiment_names(workspace_root):
+    experiment_root = Path(workspace_root) / ert3.workspace.EXPERIMENTS_BASE
+    if not experiment_root.is_dir():
+        raise ert3.exceptions.IllegalWorkspaceOperation(
+            f"the workspace {experiment_root} does not exist"
```

Comment:
This error message is kind of misleading, as the workspace probably exists, but the experiment folder does not...